<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgFileName</class>
 <widget class="QWidget" name="DlgFileName">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DlgFileName</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblToDo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Bitte geben Sie einen Namen f√ºr die zu generierende Lizenzdatei an:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="fileNameLine">
     <property name="maxLength">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblWarning">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Der Dateiname darf nicht leer sein und keines der folgenden Zeichen 
enthalten: \  / : * ? &quot; &lt; &gt; |</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
